### ISSUE TITLE
Cant manage to get it to work

### ISSUE THREAD

**Issue by calsamig** [Oskar-Idland/vscode#1]
I dont get how it is ment to work. I see some snippets that generate non-latex symbols, e.g ß. that dont appear in the pdf when I compile.  Could you provide an example of use in a latex file, and indicate what extensions or config are required?

**Comment by Oskar-Idland** [comment]
Yes, that is a LaTeX issue. You are welcome to just replace it with `\beta`, but I find it easier to read this way. I use the `unicode-math` package for this. 
![image](https://github.com/user-attachments/assets/13e6e682-ffa2-4e4d-8d0f-f20c54c77c3d)

If you install the "Fast Unicode Math Characters" extension, you can write any special character like `\beta`, then press space, and you get the actual `ß`. 

I also really recommend the "Ultra Math Preview" extension which gives you this small box which updates as fast as you type. This lets you compile less often, and spot small mistakes wayyy quicker. 

Another very useful extension is the "Conceal" extension, which lets you hide a lot of LaTeX, making it prettier and easier to read. It can be toggled if you want to debug some syntax mistake. It makes this:
![image](https://github.com/user-attachments/assets/e75a3506-1990-47f8-8d34-c4ddedbd8e78)
Become this:
![image](https://github.com/user-attachments/assets/12347656-4380-452b-bc93-6e50db02d5e9)
Where all replacements are configurable.

**Comment by calsamig** [comment]
nice! it does work smoothly with xelatex, but my docs often need to be compiled in pdflatex. So, I'll convert math unicode to latex command as you suggest.

**Comment by Oskar-Idland** [comment]
It's a long time since I used pdflatex. To reproduce the above, you can use both `ucs` with the "mathletters" option, and the `inputenc` with the "utf8x" option:
```latex
\documentclass{article}
\usepackage{physics}
\usepackage[mathletters]{ucs}
\usepackage[utf8x]{inputenc}
\author{Oskar Idland}
\title{Test Document}
\date{}
\begin{document}
\maketitle
\[
iℏ \frac{∂}{∂t} \ket{ψ} = \hat{H} \ket{ψ}
\]

\[
\left<E\right> = \bra{ψ}\hat{H}\ket{ψ} = ∫_{-∞}^{∞} ψ^{*}\hat{H}ψ \ \mathrm{d}x
\]

\end{document}
```

**Comment by calsamig** [comment]
any way to make an exception within math context? For instance I would like to be able to type a free text without triggering math snippets when adding the label, e.g.  \label{aaa}, in an equation or writting a text within an equation with \mbox{} , \mathrm{}.

**Comment by Oskar-Idland** [comment]
If a snippet has the `m`-flag, it only activates in a math context. When you have non-math context like `\text{}` or `\mathrm{}` it doesn't activate.
